Header header
uint64 vehicleId
big_actor_msgs/LatLngAlt position
float32 heading
